seed/index.js

```diff
@@ -1,12 +1,24 @@
 require("dotenv").config()
 
 const SpoofItem = require("./../api/models/spoofitem")
-const mongoose = require("mongoose")
-mongoose.connect(process.env.MONGODB_URI, {
-  useMongoClient: true
+const {mongoose} = require('./../config/mongoose')
+const Counter = require("./../api/models/counter")
+
+
```

Comment:
I don't think `SpoofItem.remove` is synchronous. The create queries below could run before this clears the DB.

Reply:
updated with promise.all